### PR TITLE
COMP: Update Azure Windows Python version to 3.11

### PR DESCRIPTION
**Pull request by thewtex**
We now require 3.9 -- 3.8 is end-of-life.

### CODE REVIEW

**Comment by thewtex** [comment]
The goal is to address failing CI.

**Comment by thewtex** [comment]
Follow-up to https://github.com/InsightSoftwareConsortium/ITK/pull/5011

**Review by hjmjohnson — APPROVED** [review]

File: Testing/ContinuousIntegration/AzurePipelinesWindowsPython.yml [hunk 1]
```diff
@@ -43,7 +43,7 @@ jobs:
 
     - task: UsePythonVersion@0
       inputs:
-        versionSpec: '3.8'
+        versionSpec: '3.11'
```
[the remark this answers is not in the document]
Comment (thewtex) [reply Testing/ContinuousIntegration/AzurePipelinesWindowsPython.yml:46]: @N-Dekker We do, but I do not think we should use Python 3.9 over 3.11. We have limited CI resources, and we want to cover as many Python versions as possible over the different platform builds.

[the remark this answers is not in the document]
Comment (N-Dekker) [reply Testing/ContinuousIntegration/AzurePipelinesWindowsPython.yml:46]: Thanks Matt, but if we want to cover as many Python versions as possible, we should test the minimum supported version on the CI.

[the remark this answers is not in the document]
Comment (blowekamp) [reply Testing/ContinuousIntegration/AzurePipelinesWindowsPython.yml:46]: For many projects I test the minimum and maximum versions.

[the remark this answers is not in the document]
Comment (thewtex) [reply Testing/ContinuousIntegration/AzurePipelinesWindowsPython.yml:46]: We want to test the minimum, maximum, and versions in between. Different versions can have different issues.

[the remark this answers is not in the document]
Comment (blowekamp) [reply Testing/ContinuousIntegration/AzurePipelinesWindowsPython.yml:46]: Can't test everything for every PR. Some things need to be tested on PR, and per complete testing on merge or nightly.

I think I am off topic now :) Hope every one has a good weekend .
